### PR TITLE
CLDR-17483 add lld Ladin locale

### PR DESCRIPTION
**Pull request by DavidLRowe**
CLDR-17483

- [ ] This PR completes the ticket.

This PR adds lld base locale; ticket asks for additional variants to be added.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by DavidLRowe** [comment]
@srl295 Can you tell me why this failed? In the log, I see:
Error:  (TestCoverageLevel.java:1282)  Error: : Language subtags that are in a CLDR locale's ID are in root (1): expected "", got "lld"

**Comment by srl295** [comment]
> @srl295 Can you tell me why this failed? In the log, I see:
> 
> Error:  (TestCoverageLevel.java:1282)  Error: : Language subtags that are in a CLDR locale's ID are in root (1): expected "", got "lld"

No idea. Files look good and we need a separate ticket to fix the error message, I can't make sense of it. @macchiati ?

**Comment by srl295** [comment]
Update: it was just attributeValueValidity.xml - and an unclear message. Filed CLDR-16818 to fix message. and updated the xml here

**Review by srl295 — DISMISSED** [review]

**Comment by DavidLRowe** [comment]
build fails with: 
```
Error:  (TestCoverageLevel.java:801)  Error: Comprehensive & no exception for path =>	//ldml/localeDisplayNames/languages/language[@type="lld"]
```

Any advice?

**Comment by srl295** [comment]
> build fails with:
> 
> ```
> Error:  (TestCoverageLevel.java:801)  Error: Comprehensive & no exception for path =>	//ldml/localeDisplayNames/languages/language[@type="lld"]
> ```
> 
> Any advice?

Looks like it needs to be added to https://github.com/unicode-org/cldr/blob/main/tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java#L527 

I reviewed that with "need to revisit next time" - can you collect a ticket with the issues found in adding a locale?  it seems like we may have actually regressed.

**Comment by srl295** [comment]
@DavidLRowe see if that does it

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java [hunk 1]
```diff
@@ -524,7 +524,7 @@ public void TestCoverageCompleteness() {
                                 + "|gaa|gag|gan|gay|gba|gbz|gez|gil|glk|gmh|goh|gom|gon|gor|got|grb|grc|gsw|guc|gur|guz|gwi"
                                 + "|hai|hak|haw|hax|hdn|hif|hil|hit|hnj|hsn|hup|hur|iba|ilo|inh|izh|jam|jbo|jgo|jmc|jpr|jrb|jut"
                                 + "|kaa|kab|kac|kaj|kam|kaw|kbd|kbl|kcg|kde|ken|kfo|kgp|kha|kho|khq|khw|kiu|kln|kmb|koi|kos|kpe|krc|kri|krj|krl|kru|ksb|ksf|ksh|kum|kut|kwk|kxv"
-                                + "|lad|lag|lah|lam|lez|lfn|lij|lil|liv|lkt|lmo|lol|lou|loz|lrc|ltg|lua|lui|lun|luo|lus|luy|lzh|lzz"
+                                + "|lad|lag|lah|lam|lez|lfn|lij|lil|liv|lkt|lld|lmo|lol|lou|loz|lrc|ltg|lua|lui|lun|luo|lus|luy|lzh|lzz"
```
Comment (srl295) [inline tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java:527]: @macchiati what's the purpose of this list? (not clear from #3230  / CLDR-17042 )

**Review by srl295 — APPROVED** [review]

**Comment by DavidLRowe** [comment]
So additional changes were needed to 
`common/supplemental/attributeValueValidity.xml`
and
`tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java`
in order to pass the tests. 

Should this information be added to the procedure document? Or are you saying that something else should change so that these additions are not needed?

**Comment by srl295** [comment]
> So additional changes were needed to `common/supplemental/attributeValueValidity.xml` and `tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java` in order to pass the tests.
> 
> Should this information be added to the procedure document? Or are you saying that something else should change so that these additions are not needed?

Add it for now but I think it's a bug.

**Pull request merged**